test/e2e/ms_pod_disruption/disruption_lib.go

```diff
@@ -53,43 +52,54 @@ func (env *DisruptionEnv) UnsuppressMayastorPod() {
 	}
 }
 
-// return the node of the replica to remove, the nodes in the
-// volume and a vector of the mayastor-hosting nodes in the cluster
-func getNodes(uuid string) (string, []string, []string) {
+// return the node of the replica to remove,
+// and a vector of the mayastor nodes not in the volume
+func getNodes(uuid string) (string, []string) {
 	nodeList, err := common.GetNodeLocs()
 	Expect(err).ToNot(HaveOccurred())
 
-	var replicaToRemove = ""
-	nexusNode, replicaNodes := common.GetMsvNodes(uuid)
-	Expect(nexusNode).NotTo(Equal(""))
+	nexus, replicaNodes := common.GetMsvNodes(uuid)
+	Expect(nexus).NotTo(Equal(""))
 
-	// find a node which is not the nexus and is a replica
+	toRemove := ""
+	// find a node which is not the nexus and is a replica (node B)
```

Comment:
What does "node B" refer to here?

Reply:
A term left over from a previous version of the code. I'll take it out next iteration.